pages/agent/v3/gcloud.md.erb

```diff
@@ -76,25 +76,37 @@ spec:
     metadata:
       labels:
         app: buildkite-agent
+      annotations:
+        "cluster-autoscaler.kubernetes.io/safe-to-evict": "true"
```

Comment:
We can probably leave this annotation out. Some folks might want it, but it's not essential and it has some sharp edges. Maybe it belongs follow up to the getting-started shaped guide?